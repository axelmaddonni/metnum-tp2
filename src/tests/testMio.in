0 0.85 0 tests/testMio.txt 0.0000000000000000000000001
