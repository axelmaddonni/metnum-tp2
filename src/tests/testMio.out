0.342466
0.131916
0.267871
0.18798
0.0348837
0.0348837
